client/landing/domains/transfer-away-confirmation/index.jsx

```diff
@@ -24,12 +22,11 @@ class TransferAwayConfirmationPage extends Component {
 
 	state = this.getLoadingState();
 
-	// @TODO: Please update https://github.com/Automattic/wp-calypso/issues/58453 if you are refactoring away from UNSAFE_* lifecycle methods!
-	UNSAFE_componentWillMount() {
+	componentDidMount() {
 		const { domain, recipientId, token } = this.props;
 		const { verifyEmail } = VerifyConfirmationCommand;
```

Comment:
What do you think about unrolling the `VerifyConfirmationCommand` constants and just passing `verify-email` string directly? Each of the constants has exactly one usage...

Or at least declaring the constants in a more traditional way than creating an object and then destructuring its fields?

Reply:
Good call! Addressed in 3d160575a81fee9ec16b5e86313ea913937a6345.